HybrasylXML/XSD/Castables.xsd

```diff
@@ -440,7 +440,6 @@
       <xs:element name="Restrictions" type="actions:Restrictions" minOccurs="0" maxOccurs="1" />
       <xs:element name="Effects" type="actions:Effects" minOccurs="0" maxOccurs="1" />
       <xs:element name="Script" type="xs:string" minOccurs="0" maxOccurs="1"/>
-      <xs:element name="Statuses" type="actions:Statuses" minOccurs="0" maxOccurs="1"/>
```

Comment:
A castable can have multiple statuses. Not sure why this was removed.

Reply:
Right now it is duplicated. This is removing this one in favor of the block under `Effects`.

Reply:
Fair enough.